include/dlaf/gpu/rocblas/error.h

```diff
@@ -21,6 +21,10 @@ namespace dlaf::gpublas::rocblas::internal {
 inline std::string getErrorString(rocblas_status st) {
   // clang-format off
   switch (st) {
+#if ROCBLAS_VERSION_MAJOR >= 3 && (ROCBLAS_VERSION_MINOR >= 1 || ROCBLAS_VERSION_MAJOR >= 4)
```

Comment:
I find the following more readable:
```suggestion
#if (ROCBLAS_VERSION_MAJOR == 3 && (ROCBLAS_VERSION_MINOR >= 1) || ROCBLAS_VERSION_MAJOR >= 4
```

Reply:
Agree, that looks much better, thanks! Updated in https://github.com/eth-cscs/DLA-Future/pull/1091/commits/a9da88766269ef0cd8d07816e8fe9150a19c5949.